src/components/Item/Item.jsx

```diff
@@ -0,0 +1,56 @@
+import defaultItemImg from "../../assets/default/default_item.png";
+import {
+  ItemImgStyle,
+  ItemNameStyle,
+  ItemPriceStyle,
+  FavoriteContainer,
+  FavoriteCountStyle,
+  FavoriteImageStyle,
+  ItemUlStyle,
+  ItemIlStyle,
+} from "./Item.styles";
+
+function ItemList({ item, variant }) {
+  const { name, price, images, favoriteCount } = item;
+
+  const handleError = (e) => {
+    e.target.src = defaultItemImg;
+  };
+
+  return (
+    <>
+      <img
+        css={ItemImgStyle(variant)}
+        src={images}
+        alt={name}
+        onError={handleError}
+      />
+      <p css={ItemNameStyle}>{name}</p>
+      <p css={ItemPriceStyle}>
+        {price.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",") + "원"}
```

Comment:
💊 제안
간단하게 아래처럼도 작성 가능합니다!
```suggestion
        {`${price.toLocaleString()} 원`}
```